app/views/shared/_header_navbar.html.erb

```diff
@@ -0,0 +1,12 @@
+<nav class="navbar navbar-expand-md navbar-dark bg-dark topbar" role="navigation">
+  <div class="<%= container_classes %>">
+    <div class="project-logo">DLME</div>
+    <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#user-util-collapse" aria-controls="user-util-collapse" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
@ggeisler Can we use the i18n `aria-label` here, similar to the [upstream partial](https://github.com/projectblacklight/spotlight/blob/master/app/views/shared/_header_navbar.html.erb)? 

We don't have this key in Spotlight's `ar.yml` yet, but hopefully we'll have it at some point in the future.
```suggestion
    <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#user-util-collapse" aria-controls="user-util-collapse" aria-expanded="false" aria-label="<%= t :toggle_nav %>">
```

Reply:
Sure, good eye @camillevilla. I started with the Blacklight version of this partial; I probably should've started with the Spotlight version.